pkg/services/rpcsrv/server.go

```diff
@@ -886,6 +887,8 @@ func (s *Server) getVersion(_ params.Params) (any, *neorpc.Error) {
 			ValidatorsCount:             byte(cfg.GetNumOfCNs(s.chain.BlockHeight())),
 			InitialGasDistribution:      cfg.InitialGASSupply,
 			Hardforks:                   hfs,
+			StandbyCommittee:            []crypto.PublicKey{cfg.StandbyCommittee},
```

Comment:
You have to explicitly convert `cfg.StandbyCommittee` strings into the list of keys, otherwise the result of the current code is a list containing a single array of strings.

Add a test to `neo-go/pkg/services/rpcsrv/server_test.go`.